lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -16,10 +17,9 @@ def initialize(serializer, options = {})
         end
 
         def serializable_hash(options = nil)
-          options = {}
```

Comment:
The implementation there was a bug, should have been `options ||= {}`. Doesn't make much sense otherwise. That's on me https://github.com/rails-api/active_model_serializers/pull/965/files#diff-ae1e38e96a8bf28ae37918a65c448c73R18